test/verifiedTests/testConnectedComponents/createToyModelForConnectedComponentAnalysis.m

```diff
@@ -0,0 +1,31 @@
+function model = createToyModelForConnectedComponentAnalysis()
+%createToyModelForConnectedComponentAnalysis creates a toy model for
+%connected component analysis
+% The model created looks as follows:
+%
+%
+%   <-> A -> B -> C <->
+%
+%
+%              -> F -
+%             /      \
+%   <-> D -> E -> G ---> H <->
+%            ^
+%            |
+%            v
+%           
+% And should yield 2 connected components, along with a bunch of exchangers
+% which are ignored.
+
+model = createModel();
+model = addReaction(model,'R1',{'A','B'},[-1 1], 0,0,1000);
+model = addReaction(model,'R2',{'B','C'},[-1 1], 0,0,1000);
+model = addReaction(model,'R3',{'D','E'},[-1 1], 0,0,1000);
+model = addReaction(model,'R4',{'E','F'},[-1 1], 0,0,1000);
+model = addReaction(model,'R5',{'E','G'},[-1 1], 0,0,1000);
+model = addReaction(model,'R6',{'F','G','H'},[-1 -1 1], 0,0,1000);
+model = addExchangeRxn(model,{'A'});
```

Comment:
I can, but honestly, a loop does not make it look any "prettier", since I would need a cell array or 2 vectors with the reaction identifiers set up. so if not necessary I would rather stick to this.